### ISSUE TITLE
Got ”RuntimeError: QUBO was not created correctly. ...” Error

### ISSUE THREAD

**Issue by 84monta** [recruit-communications/pyqubo#69]
**Describe the bug**
I got Error when I decode_sampleset with Constraint and multiplication of more than 3 variables.
Without Constraint or multiplication of 2variables, I did not get the error.

**Example Code**
```
from dwave.system import EmbeddingComposite,DWaveSampler
from pyqubo import Array,Constraint

x = Array.create('x',shape=(3),vartype="BINARY")
H = Constraint(x[0]*x[1]*x[2],label="Constraint") #RuntimeError
#H = x[0]*x[1]*x[2] #No RuntimeError

model = H.compile()
Q,Offset = model.to_qubo()

sampler = EmbeddingComposite(DWaveSampler())
responses = sampler.sample_qubo(Q,num_reads=10)

solutions = model.decode_sampleset(responses)

for sol in solutions:
    print(sol.sample)
```
**Error Message**
```
Traceback (most recent call last):
  File "XXXXXX PYTHONCODENAME", line 14, in <module>
    solutions = model.decode_sampleset(responses)
RuntimeError: QUBO was not created correctly. Please report the bug to the developer.
```

**Comment by kotarotanahashi** [comment]
@84monta Thank you for reporting the bug. I will look into it.

**Comment by kotarotanahashi** [comment]
@84monta  This problem was caused by the minor mistakes in the code. I will let you know when it's fixed. I appreciate your bug report.

**Comment by 84monta** [comment]
now this sample code seems work well. thanks!

# /usr/bin/python3 bug_test.py"
{'x[2]': 0, 'x[1]': 0, 'x[0]': 0, '0*1': 0}
{'x[2]': 1, 'x[1]': 0, 'x[0]': 0, '0*1': 0}
{'x[2]': 0, 'x[1]': 1, 'x[0]': 0, '0*1': 0}
{'x[2]': 1, 'x[1]': 1, 'x[0]': 0, '0*1': 0}
{'x[2]': 0, 'x[1]': 0, 'x[0]': 1, '0*1': 0}
{'x[2]': 1, 'x[1]': 0, 'x[0]': 1, '0*1': 0}
{'x[2]': 0, 'x[1]': 1, 'x[0]': 1, '0*1': 1}

**Comment by kotarotanahashi** [comment]
I resolved this issue in PyQUBO 1.0.4. Thank you!
https://github.com/recruit-communications/pyqubo/releases/tag/1.0.4

**Comment by 84monta** [comment]
thanks!
